sdk/src/test/java/com/hedera/hashgraph/sdk/AccountBalanceQueryTest.snap

```diff
@@ -1,8 +1,8 @@
 com.hedera.hashgraph.sdk.AccountBalanceQueryTest.shouldSerializeWithAccountId=[
-  "\ncryptoget_account_balance {\n  account_i_d {\n    account_num: 5005\n    realm_num: 0\n    shard_num: 0\n  }\n  header {\n  }\n}"
+  "# com.hedera.hashgraph.sdk.proto.Query@b01fa486\ncryptoget_account_balance {\n  account_i_d {\n    account_num: 5005\n    realm_num: 0\n    shard_num: 0\n  }\n  header {\n  }\n}"
```

Comment:
I'm pretty sure this test will be inconsistent because the object pointer is being serialized.

Reply:
Oh, that's right.  I need to implement it the same way `Tx.toString()` is implemented, and I do recall that it does something special with that.